numerics/чебышёв_series_body.hpp

```diff
@@ -47,9 +49,9 @@ Scalar ЧебышёвSeries<Scalar>::Evaluate(Instant const& t) const {
   CHECK_LE(scaled_t, 1.1);
   CHECK_GE(scaled_t, -1.1);
 
-  double b_kplus2 = 0.0;
-  double b_kplus1 = 0.0;
-  double b_k = 0.0;
+  Scalar b_kplus2{};
+  Scalar b_kplus1{};
+  Scalar b_k{};
   for (int k = degree_; k >= 1; --k) {
     b_k = coefficients_[k] + two_scaled_t * b_kplus1 - b_kplus2;
     b_kplus2 = b_kplus1;
```

Comment:
Looking at line 58 below, I think `coefficients_[0]` could lie in an affine space over the space in which the other coefficients live. We'll see about that when we get to plugging positions and vectors in there.